e2e/pages/events/draftCMOEvent.page.js

```diff
@@ -2,11 +2,22 @@ const { I } = inject();
 
 module.exports = {
   fields: {
+    recitals: {
+      title: '#recitals_0_title',
+      description: '#recitals_0_description',
+    },
     cmoHearingDateList: '#cmoHearingDateList',
   },
 
   associateHearingDate(date) {
     I.waitForElement(this.fields.cmoHearingDateList);
     I.selectOption(this.fields.cmoHearingDateList, date);
+    I.click('Continue');
+  },
+
+  enterRecital(recitals) {
+    I.click('Add new');
```

Comment:
Does `addAnotherElementToCollection`(steps_file) not work?

Reply:
done